src/Validation/Validation.php

```diff
@@ -677,7 +677,7 @@ public static function localizedTime($check, string $type = 'datetime', $format
      * @param array $booleanValues List of valid boolean values, defaults to `[true, false, 0, 1, '0', '1']`.
      * @return bool Success.
      */
-    public static function boolean($check, array $booleanValues = []): bool
+    public static function boolean(string|int|bool $check, array $booleanValues = []): bool
```

Comment:
This should probably be mixed as well. We could get anything from end user request data.